pkg/util/kubernetes/kubeadm/app/util/initsystem/initsystem_unix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
this tag is used in go1.17 or above. We're in 1.16 currently, so do not update it.